File: src/main/java/ar/edu/unq/epers/unidad4/service/interfaces/GremioService.java
package ar.edu.unq.epers.unidad4.service.interfaces;

import ar.edu.unq.epers.unidad4.model.Gremio;

public interface GremioService {
    Gremio guardar(Gremio gremio);
    Gremio recuperar(Long gremioId);
    void ingresarIntegrante(Long personajeId, Long gremioId);
    void clearAll();
}

File: src/main/kotlin/ar/edu/unq/epers/unidad4/service/GremioService.kt
package ar.edu.unq.epers.unidad4.service

import ar.edu.unq.epers.unidad4.model.Gremio

interface GremioService {
    fun guardar(gremio: Gremio): Gremio
    fun recuperar(gremioId: Long): Gremio
    fun ingresarIntegrante(personajeId: Long, gremioId: Long)
    fun clearAll()
}